index.js

```diff
@@ -63,8 +51,6 @@ const renderSubMenu = (options, param2, index) => {
     removeSubMenuElement()
```

Comment:
`param2` should be probably replaced with better name

Reply:
Solved